gcloud-workstation-base/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# As checked here https://console.cloud.google.com/artifacts/docker/cloud-workstations-images/us-central1/predefined/code-oss 
```

Comment:
Are you sure you are using pre-commit? End of file

Reply:
Will check on Monday. Committed first time from remote machine

Reply:
was not executed. Now should be ok